Bottom: 3b333948940e7dc3c722c9db4d8853f1d6244d18
Top:    99ac641d67d5f291fccd17eb9905fe57c3449c84
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:18 +0000

Turn on thindraw poly eye-candy


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 3b54c6e..8229e63 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -904,7 +904,7 @@ ghid_thindraw_pcb_polygon (hidGC gc, PolygonType *poly, const BoxType *clip_box)
 
   double old_alpha_mult = gtk_gc->alpha_mult;
   common_thindraw_pcb_polygon (gc, poly, clip_box);
-#if 0
+#if 1
   ghid_set_alpha_mult (gc, gtk_gc->alpha_mult * 0.25);
   hid_draw_fill_pcb_polygon (gc, poly, clip_box);
   ghid_set_alpha_mult (gc, old_alpha_mult);
